test/tla/cli-integration-tests.md

```diff
@@ -164,6 +164,24 @@ EXITCODE: OK
 ...
 ```
 
+### parse --output=annotations.tla Annotations succeeds
+
+```sh
+$ apalache-mc parse --output=output.tla Annotations.tla | sed 's/I@.*//'
+...
+EXITCODE: OK
+...
+```
+
+### parse --output=annotations.json Annotations succeeds
+
+```sh
+$ apalache-mc parse --output=output.json Annotations.tla | sed 's/I@.*//'
```

Comment:
Any need/desire to also check that the output.json contains the expected content?

Reply:
Yep. When Jure's PR is there, we will feed the output back to the reader ;-)